packages/build-utils/src/index.ts

```diff
@@ -116,3 +116,5 @@ export const isOfficialRuntime = (desired: string, name?: string): boolean => {
 export const isStaticRuntime = (name?: string): boolean => {
   return isOfficialRuntime('static', name);
 };
+
+export * from './workspaces';
```

Comment:
Lets be explicit with what we're exporting because this is the public interface for consumers